SDK v2 migration guide.md

```diff
@@ -91,12 +91,12 @@ The v2 strategy can be grouped into 3 categories.
     - For example, when calling device client operations before explicility calling `IotHubDeviceClient.OpenAsync()` first.
     - When some runtime validated user input is invalid in some other way, for example, importing or exporting devices where there is a null value in the list.
     - `ObjectDisposedException` when the client has been disposed but an operation has been issued.
-1. Handle canceled operations.
+2. Handle canceled operations.
   - `OperationCanceledException` when a user's cancelation token was signaled and the operation was stopped prematurely.
     > All async operations accept an optional `CancellationToken` parameter.
     > It is recommended to use cancelation tokens with a timed expiration as the SDK no longer has a default operation timeout.
     > Without that, an operation could continue indefinitely, depending on retry policy.
-1. When an operation fails due to a service error or network issues prevent issuance.
+3. When an operation fails due to a service error or network issues prevent issuance.
```

Comment:
Correcting the number

Reply:
md files actually handle the numbering automatically for you

Reply:
https://carpentries-incubator.github.io/markdown-intro/06-lists/index.html

Reply:
Weird, I am seeing [this](https://github.com/Azure/azure-iot-sdk-csharp/blob/previews/v2/SDK%20v2%20migration%20guide.md#exception-handling) 

![image](https://user-images.githubusercontent.com/94650966/230686837-23737ae9-45dd-44f3-af36-5feaae81c1fd.png)

Reply:
You're right. I wonder if that's because we have a bulleted list between the numbered list.